File: examples/mpihello.f90
program hello
implicit none
include 'mpif.h'

integer my_pe_num, ierr

call MPI_INIT(ierr)
call MPI_COMM_RANK(MPI_COMM_WORLD, my_pe_num, ierr)

print *, 'Hello from ', my_pe_num, '.'

call MPI_FINALIZE(ierr)

end program hello

File: examples/mpihello.c
#include <mpi.h>
#include <stdio.h>
int main(int argc, char **argv) {
 int rank;
 MPI_Init(&argc, &argv);
 MPI_Comm_rank(MPI_COMM_WORLD,
               &rank);
 printf("Hello from %d\n", rank);
 MPI_Finalize();
 return 0;
}
